.gitignore

```diff
@@ -80,8 +80,11 @@ typings/
 
 # Nuxt.js build / generate output
 .nuxt
-dist
-!dist/index.html
+# dist
+# !dist/index.html
+client/dist/bundle.js
+client/dist/bundle.js.map
+client/dist/bundle.js.LICENSE.txt
```

Comment:
You can also ignore all the bundle files by just adding 
bundle.*
to your .gitignore file

Reply:
Thanks! Changed.